Project/src/MakeCall/CallCard.js

```diff
@@ -892,6 +892,26 @@ export default class CallCard extends React.Component {
                     console.error(error)
                 }
             },
+            meetingAudioConferenceDetails:  async() => {
+                let messageBarText = "call in (audio only) details: \n";
+                try {
+                    const audioConferencingfeature = this.call.feature(Features.TeamsMeetingAudioConferencing);
+                    const audioConferenceDetails = 
+                        await audioConferencingfeature.getTeamsMeetingAudioConferencingDetails();
+                    console.log(`meetingAudioConferenceDetails: ${JSON.stringify(audioConferenceDetails)}`)
+                    messageBarText += `Conference Id: ${audioConferenceDetails.phoneConferenceId}\n`;
+                    if (audioConferenceDetails.phoneNumbers[0].tollPhoneNumber) { 
```

Comment:
nit: hey what if the audioconferenceDetails.phoneNumbers is null. It might throw an exception right. can we check for the length of audioConferenceDetails.phoneNumbers.length and then read the array

Reply:
I have it in try catch block so should be fine but I can include a length check in my next PR